phpcs.xml.dist

```diff
@@ -15,6 +15,13 @@
 
     <rule ref="Doctrine"/>
 
+    <!-- Disable the rules that will require PHP 7.4 -->
```

Comment:
i was disabling this (and other rules) by invoking `vendor/bin/phpcs  --config-set php_version 7020'`.. is there an equivalent command to do so in the config file?

Reply:
I didn't know you could do that, that would be great! Will look into it!

Reply:
I found the right way to express this, see the "changes" tab :)

Reply:
@morozov pointed out to me that `7020` was a bit weird, so I just pushed a new commit with `70200`, which looks more like a valid value of `PHP_VERSION`